alns/select/MABSelector.py

```diff
@@ -0,0 +1,197 @@
+import itertools
+from typing import Callable, List, Optional, Tuple, Union
+
+import numpy as np
+import pandas as pd
```

Comment:
Pandas is not an ALNS dependency - that's why the CI fails. Can we get by not typing that as a possible return value? (open question, I don't know if that's OK for MABWiser)

Reply:
Could we move this into the conditional import block?
```diff
try:
    from mabwiser.mab import ...
+    import pandas as pd
except:
   ...
```

Reply:
Yes, I just checked: pandas is a dependency for MABWiser. Let's do what you suggest!